leaflet-providers.js

```diff
@@ -121,6 +121,12 @@
 						bounds: [[46.2, -5.5], [50, 0.7]]
 					}
 				}
+				CAT: {
+					url: 'https://tile.openstreetmap.bzh/ca/{z}/{x}/{y}.png',
+					options: {
+						attribution: '{attribution.OpenStreetMap}, Tiles courtesy of <a href="https://wiki.openstreetmap.org/wiki/Ca:Mapa_en_catal%C3%A0" target="_blank">Breton OpenStreetMap Team</a>',
```

Comment:
```suggestion
						attribution: '{attribution.OpenStreetMap}, Tiles courtesy of <a href="http://www.openstreetmap.bzh/" target="_blank">Breton OpenStreetMap Team</a>',
```

Should I change the attribution link? I think that the current one is explicit enough and in the same language as the layer, so probably easier to read by the users.

Reply:
My bad, i think link should point to https://www.openstreetmap.cat, but i can't find any explicit attribution requirements for now :\

Reply:
Changed in https://github.com/leaflet-extras/leaflet-providers/pull/562/commits/c2bb9884e8894f44868353e7506d6593fd10f725